awx/main/tasks/jobs.py

```diff
@@ -1203,18 +1048,11 @@ class RunProjectUpdate(BaseTask):
 
     def __init__(self, *args, job_private_data_dir=None, **kwargs):
         super(RunProjectUpdate, self).__init__(*args, **kwargs)
-        self.playbook_new_revision = None
+        self.runner_callback = RunnerCallbackForProjectUpdate()
+        self.runner_callback.playbook_new_revision = None
```

Comment:
Setting `playbook_new_revision` to None would be better done in `RunnerCallbackForProjectUpdate.__init__`

Reply:
I have made the required changes.